### PR TITLE
Adding Help and Support page

### PR DESCRIPTION
**Pull request by sboldyreva**

### CODE REVIEW

File: docs/Help-and-Support.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+The AlmaLinux Worldwide Community is made up of users with diverse skill sets and experience levels who can assist and support you across a range of questions, topics, and troubleshooting technical issues.
+
+There are various spaces, forums and chats for discussions so that you can reach out to the Community.
+
+:::tip
+Please, note, that typically signing up is required to join AlmaLinux discussion spaces.
+:::
+
+### Chats
```
Comment (codyro) [inline docs/Help-and-Support.md:18]: > Chats

to

> Community Chat

File: docs/Help-and-Support.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+AlmaLinux Chats allows users to communicate on various channels, each with its specific topics like Packaging, Cloud, and Documentation. If you have a question or need help, post a question in the [*General*](https://chat.almalinux.org/almalinux/channels/town-square) channel or in the relevant channel for assistance.
```
Comment (codyro) [inline docs/Help-and-Support.md:20]: Proposed change(s):

> Community chats allow users to communicate in various channels, each with specific topics like Packaging, Cloud, and Documentation. If you have a question or need help, post a question in the [*General*](https://chat.almalinux.org/almalinux/channels/town-square) channel or in the relevant channel for assistance.

File: docs/Help-and-Support.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+### Forums 
+
+AlmaLinux OS also provides forums for sharing information and assistance inside the Community. Subforum based on categories for the convenience of locating a relevant one.
+
+* [AlmaLinux Forums](https://forums.almalinux.org/) 
+
+### Mailing Lists
+
+You can receive information, announcements, news letters, and updates over email by subscribing to the [AlmaLinux Mailing Lists](http://lists.almalinux.org/).
```
Comment (codyro) [inline docs/Help-and-Support.md:33]:
```
[AlmaLinux Mailing Lists](http://lists.almalinux.org/).
```

to (link to `https`)

```
> [AlmaLinux Mailing Lists](https://lists.almalinux.org/).
```

File: docs/Help-and-Support.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+### Chats
+
+AlmaLinux Chats allows users to communicate on various channels, each with its specific topics like Packaging, Cloud, and Documentation. If you have a question or need help, post a question in the [*General*](https://chat.almalinux.org/almalinux/channels/town-square) channel or in the relevant channel for assistance.
+
+* [Mattermost Community Chat](https://chat.almalinux.org/)
+* [IRC Real Time Chat](https://web.libera.chat/#almalinux)
+
+### Forums 
+
+AlmaLinux OS also provides forums for sharing information and assistance inside the Community. Subforum based on categories for the convenience of locating a relevant one.
```
Comment (codyro) [inline docs/Help-and-Support.md:27]: > AlmaLinux OS also provides public forums for free information sharing and assistance from other AlmaLinux members. Additional Subforum categories foster conversations around a specific aspect of AlmaLinux.

File: docs/Help-and-Support.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+When asking for assistance, we kindly ask you to use the mentioned below ways of sharing any output.
+
+Note, that it's preferable and recommended to use `bash` syntax.
+
+#### paste.almalinux.org
+
+We suggest you use the [https://paste.almalinux.org/](https://paste.almalinux.org/) service for sharing your outputs. 
+
+:::warning
+Please, note, that outputs shared via [https://paste.almalinux.org/](https://paste.almalinux.org/) expire 1 hour to 1 week.
```
Comment (codyro) [inline docs/Help-and-Support.md:75]: We can allow 1 day - indefinite. I just need to make sure no one has existing pastes that they need (as its stored in memory ATM). 

cc @jonathanspw

**Review by codyro — CHANGES_REQUESTED** [review]
Made some suggestions. I'm open to feedback :).

File: docs/Help-and-Support.md (hunk 5 above)
Comment (codyro) [reply docs/Help-and-Support.md:75]: This has been adjusted to allow longer expiry for pastes + persistence. You can safely remove this warning now.

image

**Comment by sboldyreva** [comment]
@codyro done, thank you for the review!

**Review by codyro — APPROVED** [review]

**Pull request merged**